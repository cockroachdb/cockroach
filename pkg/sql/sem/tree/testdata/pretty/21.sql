SELECT CAST('{"a": "b"}'::JSONB AS STRING)
