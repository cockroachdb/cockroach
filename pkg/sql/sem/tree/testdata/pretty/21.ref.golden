// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
SELECT
	CAST(
		'{"a": "b"}'::JSONB
			AS STRING
	)

37:
-------------------------------------
SELECT
	CAST(
		'{"a": "b"}'::JSONB AS STRING
	)

39:
---------------------------------------
SELECT
	CAST('{"a": "b"}'::JSONB AS STRING)

42:
------------------------------------------
SELECT CAST('{"a": "b"}'::JSONB AS STRING)


